src/src/hooks/Capabilities.js

```diff
@@ -203,9 +203,10 @@ export function useCapabilityAwsAccount(capabilityDefinition) {
   const [awsAccountInfo, setAwsAccountInfo] = useState(null);
 
   const link = capabilityDefinition?._links?.awsAccount;
+  const shouldGet = (link?.allow || []).includes("GET");
```

Comment:
I would call this "canGet" instead.
"should" makes it sound to me as if `allow.includes("GET")` is the wanted/expected behavior, whereas `.allow` not containing GET when it _should_ not is perfectly wanted behavior.

Reply:
So the frontend can fetch our awsAccount, but should it?

I think 'should' is stronger than 'can', but I am okay with changing this.
This phrasing changes the 'allow' to an order, but that may not be what we want.